wow-galaxy-mint-site/pages/api/constants.ts

```diff
@@ -0,0 +1,10 @@
+import { globals } from '../../config/globals';
+
+export default function async(_: any, res: any) {
```

Comment:
naming this async is so scuffed lmao. 
```suggestion
export default function(_req: NextApiRequest, res: NextApiResponse) {
```